src/Loggers/LogGraph.jl

```diff
@@ -0,0 +1,83 @@
+"""
+	log_graph
+"""
+function log_graph(logger::TBLogger, g::AbstractGraph; step = nothing, nodelabel::Vector{String} = map(string, vertices(g)), nodeop::Vector{String} = map(string, vertices(g)), nodedevice::Vector{String} = fill("cpu", nv(g)), nodevalue::Vector{Any} = fill(nothing, nv(g)))
+	@assert nv(g) == length(nodelabel) "length of nodelable must be same as number of vertices"
+	@assert nv(g) == length(nodeop) "length of nodeop must be same as number of vertices"
+	@assert nv(g) == length(nodedevice) "length of nodedevice must be same as number of vertices"
+	@assert nv(g) == length(nodevalue) "length of nodevalue must be same as number of vertices"
+	summ = SummaryCollection(graph_summary(g, nodelabel, nodeop, nodedevice, nodevalue))
+    write_event(logger.file, make_event(logger, summ, step=step))
+end
+
+function graph_summary(g, nodelabel, nodeop, nodedevice, nodevalue)
+	function getdtype(dtype)
+		nodetype =
+		dtype == nothing			? _DataType.DT_INVALID :
+		dtype == UInt8				? _DataType.DT_UINT8 :
+		dtype == UInt16				? _DataType.DT_UINT16 :
+		dtype == UInt32				? _DataType.DT_UINT32 :
+		dtype == UInt64				? _DataType.DT_UINT64 :
+		dtype == Int8				? _DataType.DT_INT8 :
+		dtype == Int16				? _DataType.DT_INT16 :
+		dtype == Int32				? _DataType.DT_INT32 :
+		dtype == Int64				? _DataType.DT_INT64 :
+		dtype == Float16			? _DataType.DT_BFLOAT16 :
+		dtype == Float32			? _DataType.DT_FLOAT :
+		dtype == Float64			? _DataType.DT_DOUBLE :
+		dtype <: AbstractString		? _DataType.DT_STRING :
+		dtype == Bool				? _DataType.DT_BOOL :
+		dtype ∈ [Complex{Float32},
+				Complex{Float16},
+				Complex{UInt8},
+				Complex{UInt16},
+				Complex{UInt32},
+				Complex{Int8},
+				Complex{Int16},
+				Complex{Int32}]		? _DataType.DT_COMPLEX64 :
+		dtype ∈ [Complex{Float64},
+				Complex{UInt64},
+				Complex{Int64}]		? _DataType.DT_COMPLEX128 :
+		@error "Unknown Datatype" dtype
```

Comment:
This will log a error message.
It won't throw an error.
I'm not sure if that is intended